src/sentry/api/endpoints/group_similar_issues_embeddings.py

```diff
@@ -76,4 +120,9 @@ def get(self, request: Request, group) -> Response:
             similar_issues_params.update({"threshold": float(request.GET["threshold"])})
 
         results = get_similar_issues_embeddings(similar_issues_params)
-        return Response(results)
+
+        if not results["responses"]:
+            return Response([])
```

Comment:
Do you want to track a metric or a Sentry `capture_message` to keep track of how often this happens?

Reply:
no, having an empty result set is not necessarily a sign that something isn't working - it could just be that the issue doesn't have any similar issues